ast/testcase/testcase.go

```diff
@@ -149,7 +149,7 @@ func extendTest(file *ast.File, extends string, modules TestModules) ([]*ast.Imp
 	if err != nil {
 		return nil, nil, nil, err
 	}
-	contents, err := ioutil.ReadAll(f)
+	contents, err := io.ReadAll(f)
```

Comment:
Wait did they move this function? Oh wow the entirety of ioutil is deprecated now...